File: multiplication_table.php
#!/usr/bin/php
<?php

function multiply(){
$multi_num1 = 12;
$multi_num2 = 12;
#outerloop
for($i =1; $i <= 12; $i++){
   #inner loop
    
    for ($j=1; $j <= 12; $j++){
        $ans = $j * $i;
        printf ("%4d", $ans);
    }
    print "\n";
}
}
multiply();
?>

File: multiplication_table.py
#!/us/bin/python
"""
Output the multiplcation table
"""

multi_num_1 = 13
multi_num_2 = 13

#Outer loop

for outer in range(1,multi_num_1):
    for inner in range(1, multi_num_2):
        ans = inner * outer
        print "%4d" % ans ,
    
    print ""
